app/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Metadata } from "next"
+import { Noto_Sans_KR } from "next/font/google"
+import "@styles/reset.css"
+import "@styles/color.css"
+import Header from "@/components/header/header"
+
+const NotoSansKR = Noto_Sans_KR({ subsets: ["latin"] })
+
+export const metadata: Metadata = {
+  title: "Create Next App",
+  description: "Generated by create next app",
+}
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode
+}>) {
```

Comment:
```suggestion
}: React.PropsWithChildren) {
```

이런식으로 children을 사용할수도 있습니다.
ReadOnly utility type을 사용하신 이유가 있을까요?

Reply:
공식문서에서 ReadOnly를 사용하더라구요. 그래서 저도 똑같이 사용했습니다 ㅋㅋㅋ....